apps/etterlatte-behandling/src/main/kotlin/config/ApplicationContext.kt

```diff
@@ -285,6 +289,8 @@ internal class ApplicationContext(
     val axsysKlient: AxsysKlient = AxsysKlientImpl(axsysKlient(config), url = config.getString("axsys.url")),
     val pdlTjenesterKlient: PdlTjenesterKlient = PdlTjenesterKlientImpl(config, pdlHttpClient(config)),
     val kodeverkKlient: KodeverkKlient = KodeverkKlientImpl(config, httpClient()),
+    val vvGrunnlagKlient: GrunnlagKlientVV = GrunnlagKlientImplVv(config, httpClient()),
```

Comment:
Finnes det ikke flere andre grunnlagsklienter som vilkårsvurdering kan bruke her? Om dette er for å unngå å endre noe nå i første runde så kanskje legg på en TODO for å fjerne / sammenstille